internal/c/libqb.cpp

```diff
@@ -25785,33 +25785,11 @@ void sub__font(int32 f, int32 i, int32 passed) {
     }
     im = &img[i];
     // validate f
-    i2 = 0;
-    if (f == 8)
-        i2 = 1;
-    if (f == 9) {
-        i2 = 1;
-        if (!im->text) {
-            error(5);
-            return;
-        } // font 9 can *only* be used on text surfaces
-    }
-    if (f == 14)
-        i2 = 1;
-    if (f == 15) {
-        i2 = 1;
-        if (!im->text) {
-            error(5);
-            return;
-        } // font 15 can *only* be used on text surfaces
-    }
-    if (f == 16)
+    if ((f == 8) || (f == 9) || ((f > 13) && (f < 18))) // built-in fonts 8, 9, 13, 14, 15, 16 should be valid

```

Comment:
The code looks correct, I think the comment has a typo in that `13` isn't valid and `17` is.